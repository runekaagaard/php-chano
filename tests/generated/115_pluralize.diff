001- s
001+ Fatal error: Call to undefined method DtlIter::pluralize() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/generated/115_pluralize.php on line 5