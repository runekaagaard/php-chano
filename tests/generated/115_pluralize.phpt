--TEST--
A generated testfile for the "pluralize" filter.
--FILE--
<?php
include dirname(__FILE__) . '/../../DtlIter.php';
$items = new DtlIter(array(array('input' => array (
))));
foreach ($items as $i) echo $i->input->pluralize();
--EXPECT--
