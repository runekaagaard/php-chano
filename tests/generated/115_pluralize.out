Fatal error: Call to undefined method DtlIter::pluralize() in /home/rune/workspace/php-misc/php-dtl-iterator/tests/django-generated/115_pluralize.php on line 5